### PR TITLE
Modify KBL-NUC/SDC for default build on master

### PR DESCRIPTION
**Pull request by wenlingz**
Tracked-On: #3968
Signed-off-by: wenlingz <wenling.zhang@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
OK to verify

**Comment by acrnsi** [comment]
OK to verify

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by dongyaozu** [comment]
@wenlingz this is managed by Wu Binbin. Please select right reviewer

**Comment by wenlingz** [comment]
> @wenlingz this is managed by Wu Binbin. Please select right reviewer

Thanks remind, will change it

File: misc/efi-stub/Makefile [hunk 1]
```diff
@@ -91,22 +91,11 @@ all: $(EFIBIN)
 install: $(EFIBIN) install-conf
 	install -D $(EFIBIN) $(DESTDIR)/usr/lib/acrn/$(HV_FILE).$(BOARD).$(SCENARIO).efi
 # this is to keep the compatible with original output files
```
Comment (binbinwu1) [inline misc/efi-stub/Makefile]: Is the comment still needed?

File: misc/efi-stub/Makefile [hunk 2]
```diff
@@ -91,22 +91,11 @@ all: $(EFIBIN)
...
-ifeq ($(BOARD), nuc6cayh)
-    ifeq ($(SCENARIO), sdc)
-	install -D $(EFIBIN) $(DESTDIR)/usr/lib/acrn/$(HV_FILE).efi
-    endif
-endif
 
 install-debug: $(HV_OBJDIR)/$(HV_FILE).map $(HV_OBJDIR)/$(HV_FILE).out
 	install -D $(HV_OBJDIR)/$(HV_FILE).out $(DESTDIR)/usr/lib/acrn/$(HV_FILE).$(BOARD).$(SCENARIO).efi.out
 	install -D $(HV_OBJDIR)/$(HV_FILE).map $(DESTDIR)/usr/lib/acrn/$(HV_FILE).$(BOARD).$(SCENARIO).efi.map
 # this is to keep the compatible with original output files
```
Comment (binbinwu1) [inline misc/efi-stub/Makefile]: ditto

File: misc/efi-stub/Makefile (hunk 1 above)
Comment (wenlingz) [reply misc/efi-stub/Makefile]: done, have removed

File: misc/efi-stub/Makefile (hunk 2 above)
Comment (wenlingz) [reply misc/efi-stub/Makefile]: done

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**